src/frontc/cabs2cil.ml

```diff
@@ -616,8 +616,8 @@ let alphaConvertVarAndAddToEnv (addtoenv: bool) (vi: varinfo) : varinfo =
  * elements. *)
 let rec stripConstLocalType (t: typ) : typ =
   let dc a =
-    if hasAttribute "const" a then
-      dropAttribute "const" a
+    if hasAttribute "pconst" a then
+      dropAttribute "pconst" a
     else a
```

Comment:
We could just directly call `dropAttribute`, `hasAttribute` iterates over all of it anyway.
Also does it suffice to drop `pconst` or does normal `const` also need to be dropped?

Reply:
Since this is now all for just stripping the printed consts, we don't want to do for the actual const that's needed for type comparison.
I'll do the simplification.